hydra_base/lib/scenario.py

```diff
@@ -120,17 +122,26 @@ def copy_data_from_scenario(resource_attrs, source_scenario_id, target_scenario_
 
     return target_resourcescenarios
 
-def get_scenario(scenario_id, get_parent_data=False, **kwargs):
+def get_scenario(scenario_id, get_parent_data=False, include_data=True, include_group_items=True, **kwargs):
     """
         Get the specified scenario
+        args:
+            scenario_id: The ID of the scenario to retrieve
+            include_data: Flag to indicate wheter to return the list of resource scenarios
```

Comment:
`get_parent_data` is missing from docstring.

Reply:
Also `wheter` typo.

Reply:
Done